Cargo.toml

```diff
@@ -778,29 +778,28 @@ move-vm-test-utils = { path = "third_party/move/move-vm/test-utils", features =
 move-vm-types = { path = "third_party/move/move-vm/types" }
 
 [profile.release]
+codegen-units = 1
 debug = true
+lto = "thin"
 overflow-checks = true
 
 # The performance build is not currently recommended
 # for production deployments. It has not been widely tested.
 [profile.performance]
 inherits = "release"
-opt-level = 3
 debug = true
-overflow-checks = true
-lto = "thin"
-codegen-units = 1
 
 [profile.cli]
 inherits = "release"
+lto = false
```

Comment:
why false here and off below? is that not the same?

Reply:
Previously these things that inherit from release had this setting

I was trying to leave them releasing here and not using the optimized build for now

Although if we wanna opt them all in that could work too but i think we might wanna do that separately

Definition for release profile defaults below

https://doc.rust-lang.org/beta/cargo/reference/profiles.html#release